### PR TITLE
change link to telesec certificate

### PR DESCRIPTION
**Pull request by chrbeckm**

### CODE REVIEW

File: faq/index.md [hunk 1]
```diff
@@ -21,7 +21,7 @@ Um das eduroam-Netz an der TU Dortmund einzurichten, können folgende Optionen v
  - Domain: tu-dortmund.de
  - CA-Zertifikat: T-TeleSec_GlobalRoot_Class_2.crt
     - zu finden in `/usr/share/ca-certificates/mozilla/T-TeleSec_GlobalRoot_Class_2.crt`
```
Comment (maxnoe) [inline faq/index.md:23]: Auf Ubuntu 22.04 finde ich es unter: `/etc/ssl/certs/T-TeleSec_GlobalRoot_Class_2.pem`

File: faq/index.md [hunk 2]
```diff
@@ -12,16 +12,17 @@ Am Ende der Installationsanleitungen gibt es den Punkt
...
 # Linux
 
-## <a id="w-lan"></a>W-LAN unter Ubuntu
+## <a id="w-lan"></a>W-LAN unter Ubuntu/Linux
 
 Um das eduroam-Netz an der TU Dortmund einzurichten, können folgende Optionen verwendet werden:
  - Wi-Fi security: WPA- & WPA2-Enterprise
  - Authentication: Geschütztes EAP (PEAP)
- - Anonymous Identity:  
+ - Anonymous Identity:
```
Comment (maxnoe) [inline faq/index.md:20]: This should be `telesec@tu-dortmund.de` according to https://service.tu-dortmund.de/ubuntu-wlan

Comment (chrbeckm) [reply faq/index.md:20]: For me that field is empty, do you have the site cached?

Comment (maxnoe) [reply faq/index.md:20]: The screenshots don't match the text... and the Ubuntu 18 don't match the Ubuntu 12 settings, although there should be no difference between the two for the basic wifi settings of the network... 

I remember needing the anonymous identity

**Review by maxnoe — APPROVED** [review]

**Pull request merged**